client/src/components/Donation/donate-form.tsx

```diff
@@ -26,14 +26,14 @@ import {
   postChargeStripeCard
 } from '../../redux';
 import Spacer from '../helpers/spacer';
-import DonateCompletion from './DonateCompletion';
+import DonateCompletion from './donate-completion';
 import PatreonButton from './patreon-button';
 import type { AddDonationData } from './paypal-button';
 import PaypalButton from './paypal-button';
 import StripeCardForm, { HandleAuthentication } from './stripe-card-form';
 import WalletsWrapper from './walletsButton';
 
-import './Donation.css';
+import './donation.css';
```

Comment:
Looks like either the file needs to be renamed or this change needs to be reverted.

Reply:
I renamed that file, but git somehow did not pick it up. :)